closed/OpenJ9.gmk

```diff
@@ -365,7 +365,10 @@ $(J9JCL_SOURCES_DONEFILE) : \
 	@$(ECHO) Generating J9JCL sources
 	$(call RunJPP, GENERIC, $(TOPDIR)/closed/adds/jdk/src/share/classes, $(JPP_DEST)) \
 		$(IncludeIfUnsure)
-	$(call RunJPP, SIDECAR18-SE-OPENJ9, $(OPENJ9_TOPDIR)/jcl, $(JPP_DEST))
+	$(call RunJPP, SIDECAR18-SE-$(if $(CLOSEDJ9_SHA),CLOSED,OPEN)J9, $(OPENJ9_TOPDIR)/jcl, $(JPP_DEST))
+  ifneq (,$(CLOSEDJ9_SHA))
+	$(call RunJPP, SIDECAR18-SE-CLOSEDJ9SRC, $(CLOSEDJ9_TOPDIR)/jcl, $(JPP_DEST))
+  endif
```

Comment:
Since we need `ifneq` anyway, suggest this is clearer as:
```
  ifneq (,$(CLOSEDJ9_SHA))
	$(call RunJPP, SIDECAR18-SE-CLOSEDJ9, $(OPENJ9_TOPDIR)/jcl, $(JPP_DEST))
	$(call RunJPP, SIDECAR18-SE-CLOSEDJ9SRC, $(CLOSEDJ9_TOPDIR)/jcl, $(JPP_DEST))
  else
	$(call RunJPP, SIDECAR18-SE-OPENJ9, $(OPENJ9_TOPDIR)/jcl, $(JPP_DEST))
  endif
```